src/test/java/support/matchers/ResponseMatchers.java

```diff
@@ -53,17 +105,21 @@ protected boolean matchesSafely(Response response) {
       @Override
       public void describeTo(Description description) {
         description
-          .appendText("Response has 422 status and 'message' - ").appendValue(expectedMessage)
+          .appendText("Response has 422 status and 'message' - ").appendValue(
+            expectedMessage)
           .appendText(", 'key' - ").appendValue(expectedKey)
           .appendText(" and 'value' - ").appendValue(expectedValue);
       }
 
       @Override
-      protected void describeMismatchSafely(Response response, Description mismatchDescription) {
-        mismatchDescription.appendText("Status: ").appendValue(response.getStatusCode())
+      protected void describeMismatchSafely(Response response,
+        Description mismatchDescription) {
+        mismatchDescription.appendText("Status: ").appendValue(
+            response.getStatusCode())
           .appendText(", body: ");
 
-        if (response.getContentType().startsWith(ContentType.APPLICATION_JSON)) {
+        if (response.getContentType().startsWith(
+          ContentType.APPLICATION_JSON)) {
```

Comment:
As this is too long for a single line, I'd be tempted to extract a method like `isJsonContent`

Reply:
@marcjohnson-kint agree. A brand new method was added.